testdata/serviceconfig.yml

```diff
@@ -31,7 +31,18 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
```

Comment:
Both work, updated to use camel case as more idiomatic yaml.

This is gonna be tricky for our users down the line: `labels` (`map[string]string`) look the same as normal fields in yaml but _must_ match the label names exactly (so underscores matter there).